src/config.ts

```diff
@@ -39,14 +40,19 @@ export interface SnowpackSourceFile {
   locOnDisk: string;
 }
 
-export interface BuildOptions {
-  contents: string;
+export interface LoadOptions {
   filePath: string;
   fileExt: string;
   isDev: boolean;
   log: (msg, data) => void;
-  /** DEPRECATED */
```

Comment:
Should this property still be listed since we're including in code above? Maybe just adding `@deprecated`?

Reply:
I think by deprecating it we want to remove it from the documented interface, especially since people will be looking at this to decide how to migrate their existing plugins.